noir_stdlib/src/prelude.nr

```diff
@@ -1,3 +1,6 @@
 use crate::collections::vec::Vec;
 use crate::option::Option;
 use crate::{print, println, assert_constant};
+use crate::cmp::{Eq, Ord, Ordering};
+use crate::default::Default;
+use crate::ops::{Add, Sub, Mul, Div, Rem, BitOr, BitAnd, BitXor, Shl, Shr};
```

Comment:
These aren't imported by default in Rust, do we need these to be imported automatically?

Reply:
We don't need them, although I thought they'd be useful. I can remove them if we want to stay closer to rust in this regard.

Reply:
I think we can remove these from the prelude. It's more rust-like and ideally we keep the prelude as slim as possible.

Reply:
Done